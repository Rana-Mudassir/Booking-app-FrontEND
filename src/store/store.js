import { configureStore } from '@reduxjs/toolkit';
import loginReducer from './authentication/login';
import signupReducer from './authentication/signup';
import aeroplaneReucer from './aeroplane/aeroplane';
import aeroplaneslistReducer from './aeroplaneList/aeroplaneListSlice';
import selectedAirplaneSliceReducer from './aeroplaneList/selectedAirplaneSlice';

const store = configureStore({
  reducer: {

    login: loginReducer,
    signup: signupReducer,
    aeroplane: aeroplaneReucer,
    aeroplaneslist: aeroplaneslistReducer,
    selectedAirplane: selectedAirplaneSliceReducer,
  },
});

export default store;
