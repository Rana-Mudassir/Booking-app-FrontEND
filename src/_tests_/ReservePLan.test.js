import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../store/store';
import ReservePLan from '../components/pages/reserve/ReservePLane';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <Router>
          <ReservePLan />
        </Router>
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
