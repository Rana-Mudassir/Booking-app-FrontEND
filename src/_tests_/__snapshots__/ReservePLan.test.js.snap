// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="reservattionSection"
>
  <h2
    className="reservationH"
  >
    Reserve Aeroplane
  </h2>
  <p
    className="reservationP"
  >
    Please fill the form to reserve your Aeroplane
  </p>
  <form
    className="formContainer"
  >
    <label
      className="labelText"
      htmlFor="name"
    >
      Name:
      <input
        id="name"
        onChange={[Function]}
        type="text"
        value=""
      />
    </label>
    <br />
    <label
      className="labelText"
      htmlFor="city"
    >
      City:
      <input
        id="city"
        onChange={[Function]}
        type="text"
        value=""
      />
    </label>
    <br />
    <label
      className="labelText"
      htmlFor="date"
    >
      Rervation Date:
      <input
        id="date"
        onChange={[Function]}
        type="date"
        value=""
      />
    </label>
    <br />
    <label
      className="labelText"
      htmlFor="returndate"
    >
      Return Date:
      <input
        id="returndate"
        onChange={[Function]}
        type="date"
        value=""
      />
    </label>
    <br />
    <label
      className="labelText"
      htmlFor="aeroplane"
    >
      Aeroplane:
      <select
        id="aeroplane"
      >
        <option>
          Select Aeroplane
        </option>
        <option
          value="A320"
        >
          A320
        </option>
        <option
          value="A380"
        >
          A380
        </option>
      </select>
    </label>
    <br />
    <div
      className="btnContainer"
    >
      <button
        id="reservationBtn"
        type="submit"
      >
        Reserve
      </button>
      <button
        id="checkReservations"
        type="submit"
      >
        Check My Reservations
      </button>
    </div>
  </form>
</section>
`;
